Makefile

```diff
@@ -89,6 +89,12 @@ build: ## build application
 	@ $(GO) version
 	@ source $(ENVFILE) && $(GO) build -ldflags "$(LDFLAGS)" -o ./bin/$(NAME) ./cmd/$(BIN)
 
+.PHONY : builddev
```

Comment:
I don't see the point of creating this 🤔

Reply:
it is to facilitate my development